.env.development

```
@@ -1,2 +1,4 @@
 INLINE_RUNTIME_CHUNK=false
 PUBLIC_URL=http://localhost:3000
+LOADER_URL=https://cdn.fromdoppler.com/loader/v1/loader.js
+STYLE_GUIDE_MANIFEST_URL=https://cdn.fromdoppler.com/doppler-style-guide/asset-manifest-v1.json
```

What do you think about using main version by default?

```suggestion
STYLE_GUIDE_MANIFEST_URL=https://cdn.fromdoppler.com/doppler-style-guide/asset-manifest-main.json
```

---

Ok, just in development

---

🚀